arviz/utils.py

```diff
@@ -664,3 +665,32 @@ def credible_interval_warning(credible_interval, hdi_prob):
 
     hdi_prob = credible_interval
     return hdi_prob
+
+
+class HtmlTemplate:
+    """Contains html templates."""
```

Comment:
Expand docstring. Contains html templates for what?

Would change this to something along the lines of contains html templates for InferenceData repr